### PR TITLE
Fix potential liquity module errors

### PR DESCRIPTION
**Pull request by yabirgb**
Avoid `KeyError` and `DivisionByZero` errors in liquity module

### CODE REVIEW

**Comment by yabirgb** [comment]
Failing tests come from a 500 error calling etherscan API https://github.com/rotki/rotki/pull/3391/checks?check_run_id=3459456089#step:8:1282

File: rotkehlchen/tests/api/test_liquity.py [hunk 1]
```diff
@@ -129,3 +129,36 @@ def test_trove_events(rotkehlchen_api_server):
...
+        start_with_valid_premium,
+        rotki_premium_credentials,
+        inquirer,  # pylint: disable=unused-argument
+):
+    """Test that we can get the status of the trove and the staked lqty"""
+    rotki = rotkehlchen_api_server.rest_api.rotkehlchen
+    # Set module premium is required
+    premium = None
+    if start_with_valid_premium:
+        premium = Premium(rotki_premium_credentials)
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_liquity.py:149]: What are you doing here? Why do this?

You want this here:
https://github.com/rotki/rotki/blob/1a3c40b147054eca6b1514feff9d138028ca42e8/rotkehlchen/tests/fixtures/rotkehlchen.py#L161-L164

And don't do it just for liquity. Check all modules of the chain manager (I think they all got premium, right?) and pass its value if it's not None.

File: rotkehlchen/tests/api/test_adex.py [hunk 2]
```diff
@@ -110,12 +105,8 @@ def test_get_events(
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
 
     # Set module premium is required for calling `get_balances()`
-    premium = None
-    if start_with_valid_premium:
-        premium = Premium(rotki_premium_credentials)
-
     adex = rotki.chain_manager.get_module('adex')
-    adex.premium = premium
+    adex.premium = rotki.chain_manager.premium
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_adex.py:109]: No no no this is not what I said. What part of [what I said](https://github.com/rotki/rotki/pull/3391#discussion_r698344148) was not clear? I even point to you where in the code to change this.

Sure, you should remove it from all these places you removed  it BUT you should instead move the instantiation to the fixture creation. At the exact point my comment links you to. You can loop through the modules of the chain manager there and pass the premium object.

**Comment by yabirgb** [comment]
The failing tests failed due to etherscan rate limiting the queries. Tried locally and works fine

**Review by LefterisJP — APPROVED** [review]
LGTM. Thanks for making that cool refactor on the tests too.

**Pull request merged**